doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,120 @@
+# How we Improved the Files Virtual Schema Performance
+
+Our [Virtual Schemas for document files](https://github.com/exasol/virtual-schema-common-document-files) allow our customers to integrate formatted data from files into their Exasol database. Our customers use this feature for integrating external data on demand into their analytics calculation and for building Extract, Transform and Load (ETL) pipelines.
+
+For Exasol speed matters. So does it for our integrations. For customers, it's crucial that the queries accessing external data run as fast as possible. In this blog post I want to give you some insights from our recent performance improvements of the Virtual Schema for [Apache Parquet](https://parquet.apache.org/) files.
+
+## How it Begins
+
+For me, this project started when our product manager asked me to improve the performance of the Parquet file loading. He compared the performance of our products with other solutions. As a result of this comparison, we realized that our Parquet loading needs major improvements.
```

Comment:
good story telling, but not sure if we should be that open

Reply:
I wasn't sure myself and checked with Thmas.